docs/sources/configuration/_index.md

```diff
@@ -746,10 +746,14 @@ results_cache:
 The `ruler` block configures the Loki ruler.
 
 ```yaml
-# URL of alerts return path.
+# URL base of the explore page.
```

Comment:
Is that right though? We want them to just define the base URL for the Grafana instance. We add in `/explore`, do we not?

Reply:
Yeah that's what I wanted to say but I keep coming up with misleading phrases :smile: 
How about this: `Base URL of the Grafana instance.` ?

Reply:
👌 perfect